src/features/ImageGridWidget/ImageGridWidget.constants.ts

```diff
@@ -0,0 +1,8 @@
+export const IMAGE_GRID_MOBILE_COLS = 1;
+export const IMAGE_GRID_DESKTOP_COLS = 3;
+
+export const IMAGE_GRID_MOBILE_ROW_HEIGHT = 120;
+export const IMAGE_GRID_DESKTOP_ROW_HEIGHT = 240;
+
+export const IMAGE_GRID_MOBILE_IMAGE_COUNT = 3;
+export const IMAGE_GRID_DESKTOP_IMAGE_COUNT = 4;
```

Comment:
We can group them instead of individually exporting each value.

```suggestion
export const IMAGE_GRID_CONFIG = {
  MOBILE: {
    COLS: 1,
    ROW_HEIGHT: 120,
    IMAGE_COUNT: 3,
  },
  DESKTOP: {
    COLS: 3,
    ROW_HEIGHT: 240,
    IMAGE_COUNT: 4,
  },
};
```